scripts/build.js

```diff
@@ -156,6 +153,34 @@ function build(options) {
   });
 }
 
+function resolveNodeGyp(options) {
+  var args = ['rebuild', '--verbose'].concat(
+    ['libsass_ext', 'libsass_cflags', 'libsass_ldflags', 'libsass_library'].map(function (subject) {
+      return ['--', subject, '=', process.env[subject.toUpperCase()] || ''].join('');
+    })).concat(options.args);
+
+  // For node-chakracore, check if node-gyp is in the path.
+  // If yes, use it instead of using node-gyp directly from
+  // node_modules because the one in node_modules is not
```

Comment:
Is it just a matter of bumping to a newer version in [package.json](https://github.com/sass/node-sass/blob/master/package.json#L68) or are users expected to have a custom version that only ships with node-chakracore?